src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
I don't follow this comment, sorry. So "general" is still a valid module? How is CCM:TextRender invoked since that depends on this class, not the other way? 

<guessing>Is there some magic going on related to the use of any other value passed to module causes it to look for a TT file of that name, and we always provide a general.tt file? If so, I think it would be clearer to remove this parenthesised "afterthought" sentence and rewrite line 160 to explain how general is a special case of the TT functionality.